tooling/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj

```diff
@@ -314,12 +311,19 @@
     </Reference>
     <Reference Include="WindowsBase" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <VSCTCompile Include="RazorPackage.vsct">
```

Comment:
The VSCT file handles commands and menu bindings, rather than write another one without our menu item I'm just leaving it out.